### DISCUSSION TITLE
Which conv layer should be selected as the last conv layer in gradcam example when using resnet50v2?

### DISCUSSION THREAD

**Discussion by 7oud** [apache/incubator-mxnet#13863] [Q&A]
## Description
I followed the "Visualizing Decisions of Convolutional Neural Networks", and it gives the correct output images when using vgg16 pretrained model. ThenI changed the network to ResNet50v2 with its pretrained model, but the output images looks abnormal, Some code snippet are as follow.

```
    # ResNetV2 using gradcam's Conv2D and Activation
    net = ResNetV2(BottleneckV2, layers, channels, **kwargs)
    net.initialize(ctx=ctx)

    resnet50v2 = mx.gluon.model_zoo.vision.resnet50_v2()
    # load pretrain model
    resnet50v2.load_parameters('D:/Model/mxnet/models/resnet50_v2-ecdde353.params', ctx=ctx)
    params = resnet50v2.collect_params()
    for key in params:
        param = params[key]
        net.collect_params()[net.prefix + key.replace(resnet50v2.prefix, '')].set_data(param.data())

    # ...
    last_conv_layer_name = net.features[8][2].conv3.name
    show_images(*visualize(network, "hummingbird.jpg", last_conv_layer_name))
```
![figure_1](https://user-images.githubusercontent.com/7981826/51075473-f0c09780-16c6-11e9-896e-426f395666bd.png)

The upper row uses resnet pretrained model, and the lower row uses vgg16 model

**Comment by piyushghai** [comment]
For questions, please submit them on MXNet discussion forum (https://discuss.mxnet.io), where it will get a wider audience and allow other to learn as well.
Thanks!

@mxnet-label-bot add [Question, Python, Model Zoo]
